backend/tests/unit/agents/iteration_controllers/test_forced_conclusion.py

```
@@ -318,37 +306,34 @@ async def test_forced_conclusion_with_successful_llm_response(
-                        "Based on the investigation so far, here is my conclusion: The alert indicates a potential issue."
-                    )
-                else:
-                    updated_conversation.append_assistant_message("Thought: Investigating...")
-                
-                return updated_conversation
+        # Configure agent with forced conclusion enabled
+        sample_context.agent.get_force_conclusion = Mock(return_value=True)
+        
+        # Mock conclusion call
+        call_count = 0
+        async def mock_generate_with_conclusion(conversation, session_id, stage_execution_id=None, **kwargs):
```

Ruff (0.14.11)

Unused function argument: `session_id`

(ARG001)